pkg/utils/nodeclass/suite_test.go

```diff
@@ -247,6 +247,74 @@ var _ = Describe("NodeClassUtils", func() {
 		ExpectSecurityGroupStatusEqual(nodeTemplate.Status.SecurityGroups, nodeClass.Status.SecurityGroups)
 		ExpectAMIStatusEqual(nodeTemplate.Status.AMIs, nodeClass.Status.AMIs)
 	})
+	It("should convert a AWSNodeTemplate to a EC2NodeClass (with AMISelector name and owner values set) with spaces", func() {
```

Comment:
This is probably going to get removed soon anyways, since we removed the v1alpha1 controllers/utils for now. Maybe better lived in the karpenter-convert tests?

Reply:
I was thinking this would be for the karpenter internal conversion, and I'm not sure if this would apply after we remove v1alpha1